services/e2b_sandbox.py

```
@@ -0,0 +1,408 @@
+        authed_url = remote_url
+        if token and remote_url.startswith("https://github.com/"):
+            authed_url = remote_url.replace(
+                "https://github.com/", f"https://{token}@github.com/"
+            )
+        if authed_url != remote_url:
+            try:
+                await sb.commands.run(
+                    f"git -C repo remote set-url origin {authed_url}", timeout=10
+                )
+            except Exception:  # pragma: no cover - best-effort
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)